src/shorthands/JShape.jl

```diff
@@ -0,0 +1,21 @@
+"""
+    JShape(body...)
+
+Creates a custom shape based on the luxor instructions in the begin...end block
+```julia
+somepath1 = Object(@JShape begin
+                                    sethue(color)
+                                    poly(points, action, close= true)
+                                end action = :stroke color ="red" radius = 8 
+)
+```
```

Comment:
Could you run JuliaFormatter on the code in this example? Somehow seems a bit off to me.